### DISCUSSION TITLE
Python projects with AI

### DISCUSSION THREAD

**Discussion by mansourshebli** [community/community#68192] [Programming Help] [answered]
Are there instances of practical integration of Artificial Intelligence (AI) and Machine Learning (ML) within Python-driven projects? If so, could you kindly offer a curated collection of educational materials and share exemplar project ideas for developers keen on delving into this realm?

Note: I’ve done some Python AI projects but I need to go in depth.

**Comment by mantasu** [comment]
Hey, I would suggest checking [Coursera](https://www.coursera.org/) for available projects. Not only projects, you can also find courses that teach how to build them, for example, [Developing AI Applications on Azure](https://www.coursera.org/learn/developing-ai-applications-azure/).

**Comment by levintech** [comment, marked as answer]
Hey, I think the Kaggle is best platform to learn basic theory, practice yourself and also compete with others.
